tools/Config.mk

```diff
@@ -548,3 +548,11 @@ else
   ARCHXXINCLUDES += ${shell $(INCDIR) -s "$(CC)" $(TOPDIR)$(DELIM)include$(DELIM)cxx}
 endif
 ARCHXXINCLUDES += ${shell $(INCDIR) -s "$(CC)" $(TOPDIR)$(DELIM)include}
+
+# Linker Scripts
+
+ifeq ($(CONFIG_CYGWIN_WINTOOL),y)
+  ARCHSCRIPT_CLI = $(addprefix -T,$(foreach SCRIPT,$(ARCHSCRIPT),${shell cygpath -w $(SCRIPT)}))
+else
+  ARCHSCRIPT_CLI = $(addprefix -T,$(ARCHSCRIPT))
+endif
```

Comment:
Is it possible just
```
ifeq ($(CONFIG_CYGWIN_WINTOOL),y)
  LDFLAGS = $(addprefix -T,$(foreach SCRIPT,$(ARCHSCRIPT),${shell cygpath -w $(SCRIPT)}))
else
  LDFLAGS = $(addprefix -T,$(ARCHSCRIPT))
endif
```
And drop `ARCHSCRIPT_CLI`?

Reply:
yeah, that should work. Thanks for the suggestion!

Reply:
I noticed the flags were getting double added. I added a guard around it to prevent it from ever happening. Let me know what you think